lib/sprinkler_web/live/dashboard_live.ex

```diff
@@ -0,0 +1,30 @@
+defmodule SprinklerWeb.DashboardLive do
+  use SprinklerWeb, :live_view
+
+  @telemetry_topic "telemetry"
+
+  @impl true
+  def mount(_params, _session, socket) do
+    Phoenix.PubSub.subscribe(Sprinkler.PubSub, @telemetry_topic)
+    {:ok, assign(socket, :devices, [%{id: "1", tmps: [2]}, %{id: "2", tmps: [2]}])}
+  end
+
+  def handle_info(
+        %{
+          topic: @telemetry_topic,
+          event: "new_reading",
+          payload: %{payload: payload, device_id: device_id}
+        },
+        socket
+      ) do
+    tmp = payload["tmp"]
+
+    updated_list =
+      Enum.map(socket.assigns.devices, fn
```

Comment:
Is it possible that we need to add new devices to this structure as well? I mean if you receive a `device_id` that does no exist in the structure then you need to add it with its tmp list. But I don't know if will have a situation like that.

Reply:
I think we won't, all devices should be registered by the time the page loads. We can consider the case later but we'll just have two devices. Plus in this way we just ignore telemetry for devices that don't exist so it's cool